index.src.html

```diff
@@ -635,25 +641,50 @@ <h2 id="integration-with-html">Integration with HTML</h2>
   <h2 id="integration-with-css">Integration with CSS</h2>
 
   The CSS Standard does not specify how it fetches resources referenced from
-  stylesheets. However, implementations use the URL a given stylesheet was
-  loaded from as referrer for the requests for resources from that stylesheet.
+  stylesheets. However, implementations should be sure to set the
+  referrer-related properties of any <a for="/">requests</a> initiated by stylesheets
+  as follows:
 
-  Implementations should keep track of a referrer policy for each stylesheet
-  that should be used to create requests for resources from the respective
-  stylesheet.
+  <ol>
+    <li>
+       If a <a>CSS declaration block</a> is responsible for the request,
```

Comment:
Tiny nit: If you enable markdown via `Markup Shorthands: markdown on`, you could write this as:

```
1.   Item 1.
2.   Item 2.
3.   Item 3.
```

Which is prettier! And easier to read! But has no normative impact at all so can easily be done in a future PR!